generics/middleware/authenticator.js

```diff
@@ -98,7 +98,8 @@ module.exports = async function (req, res, next) {
     return
   }
 
-  if (req.path.includes("keywords")) {
+  if (req.path.includes("keywords") || req.path.includes("/aws/uploadFile") || 
+  req.path.includes("/gcp/uploadFile") || req.path.includes("/azure/uploadFile")) {
```

Comment:
@rakeshSgr - Make this as an array and search in that.

Reply:
resolved